src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCasePreviewService.java

```diff
@@ -139,91 +141,113 @@ protected NoticeIssuedTemplateBody createPayload(PreSubmitCallbackResponse<SscsC
         return builder.build();
     }
 
+    private void handleOralHearing(Adjournment adjournment, AdjournCaseTemplateBodyBuilder adjournCaseBuilder) {
+        if (adjournment.getNextHearingListingDuration() != null) {
+            if (adjournment.getNextHearingListingDurationUnits() == null) {
+                throw new IllegalStateException("Timeslot duration units not supplied on case data");
+            }
+            adjournCaseBuilder.nextHearingTimeslot(getTimeslotString(adjournment.getNextHearingListingDuration(),
+                    adjournment.getNextHearingListingDurationUnits()));
+        } else {
+            adjournCaseBuilder.nextHearingTimeslot("a standard time slot");
+        }
+    }
+
+    private void handleFaceToFaceHearing(Adjournment adjournment, AdjournCaseTemplateBodyBuilder adjournCaseBuilder, String venueName) {
+        if (adjournment.getNextHearingVenue() == AdjournCaseNextHearingVenue.SOMEWHERE_ELSE) {
+            if (adjournment.getNextHearingVenueSelected() != null
```

Comment:
Feel like this could have used an optional.ofnullable or adjournment.getNextHearingVenueSelected() could have been a variable for this method